pi3d/Display.py

```diff
@@ -560,10 +560,10 @@ def update(self):
     display.width = display.right = display.max_width = display.opengl.width #not available until after create_display
     display.height = display.bottom = display.max_height = display.opengl.height
     display.top = display.bottom = 0
-    if frames_per_second is not None:
+    if frames_per_second:
       display.android.frames_per_second = frames_per_second
-      display.frames_per_second = None #to avoid clash between two systems!
-    
+      display.frames_per_second = 0 #to avoid clash between two systems!
```

Comment:
That comment is a bit terse (though I haven't looked at this codebase in quite a while).  

You clearly have gained significant understanding about the interaction between `display.frames.per_second` and `display.android.frames.per_second` - I don't suppose I could convince you to write a couple of lines about it, perhaps right here?

Reply:
That comment is not from me :)